newrelic/common/system_info.py

```diff
@@ -349,7 +349,6 @@ def gethostname(use_dyno_names=False, dyno_shorten_prefixes=()):
     """
 
     global _nr_cached_hostname
-    global _nr_cached_hostname_lock
```

Comment:
Why remove these?

Reply:
[PLW0602 - global-variable-not-assigned](https://docs.astral.sh/ruff/rules/global-variable-not-assigned/)

There's no need to make this variable global when it's never assigned to. It already reads the globally scoped variable when attempting to use the lock.